handler.go

```diff
@@ -145,6 +145,16 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, u *User) {
 		return
 	}
 
+	// If authentication is enabled and there are no users yet, make sure
+	// the first statement is creating a new cluster admin.
+	if h.AuthenticationEnabled && h.server.UsersLen() == 0 {
```

Comment:
Ah, ok, I see it.  I might change the signature on NewServer to take that as a parameter.  It would be more consistent to see that in a new than to see it later (as it could get easily missed if someone changes code later on).